### DISCUSSION TITLE
Windows Terminal v1.19.10302.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#16632] [Announcements]
This is a quick update to fix a blocking issue in 1.19.

## Bug Fixes

* `wt $COMMAND` no longer blows away the entire process environment, and it now works properly (#16624)

---

_Binary files inside the unpackaged distribution archive bear the version number `1.19.240130002`._

### Asset Hashes

- Microsoft.WindowsTerminal_1.19.10302.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `0993CBA7A53DA6EA4CF5F1702440E61404E9AE9463EFD5CBDC16E6ADFC5465E7`
- Microsoft.WindowsTerminal_1.19.10302.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `29F780BD04AE5F01D3EBDB63CC98A917C676AC7810DAC00D635B78F8ACD47E14`
- Microsoft.WindowsTerminal_1.19.10302.0_arm64.zip
   - SHA256 `9A3B3EF52F887518FB58AB79D14F54AB177F4D7FD42DE5C37C921E30E7A9D259`
- Microsoft.WindowsTerminal_1.19.10302.0_x64.zip
   - SHA256 `899B60D9145FCE8CE7B18C0E53A80C2362FA67C3EAB5D0823543041A7E10F0D3`
- Microsoft.WindowsTerminal_1.19.10302.0_x86.zip
   - SHA256 `27DCB0951CD83C7551F7B3EC3E997A1928D395D985015833ED11CA9976F5C48A`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.19.10302.0'>Windows Terminal v1.19.10302.0</a>.</em>

**Comment by DHowett** [comment]
:mega: This release is being rolled out to Beta and Release Preview customers today!

**Comment by aiwonderland** [comment]
Wow!

**Comment by maddes8cht** [comment]
@DHowett this should be removed from pinned discussions as it is outdated.

**Reply by DHowett** [reply]
I didn't even realize I had pinned it. Thanks!
